interface/smart/register-app.php

```diff
@@ -160,7 +160,7 @@ function registerApp() {
                     document.querySelector(".errorResponse").classList.add("hidden");
                     document.querySelector("#clientID").value = resultJSON.client_id;
                     document.querySelector("#clientSecretID").value = resultJSON.client_secret;
-                    msgText = js_xl('Your app has been registered!');
+                    msgText = window.top.xl('Your app has been registered!');
```

Comment:
looks good, but looks like some other custom stuff got in here.

Reply:
lol, great catch, I'll back out the file.